image_test/sbom/test_sbom_export.sh

```diff
@@ -0,0 +1,36 @@
+URL="http://metadata/computeMetadata/v1/instance/attributes"
+SBOM_FILE_NAME=$(curl -f -H Metadata-Flavor:Google ${URL}/sbom-file-name)
+OUTS_PATH=$(curl -f -H Metadata-Flavor:Google ${URL}/outs-path)
+DISK_FILE_NAME=$(curl -f -H Metadata-Flavor:Google ${URL}/disk-file-name)
+
+GCS_PATH_SBOM=${OUTS_PATH}/${SBOM_FILE_NAME}
+GCS_PATH_OUTDISK=${OUTS_PATH}/${DISK_FILE_NAME}
+# check if tar.gz file is there
+# check if sbom file is there
+# potentially run other script for sbom contents
+# to check, need a convention for the sbom json file name as well, input variable. 
+gsutil -q stat $GCS_PATH_SBOM
```

Comment:
Here again, I wouldn't block this PR on this but you can simplify all this check into a single line like:

```bash
gsutil -q stat $GCS_PATH_SBOM 2>&1 >> /dev/null | echo "SBOM Failed: sbom file not found" & exit 1
```

Reply:
Could you explain what the 2>&1 >> /dev/null part is doing? I tried to replace the check with this, but it seemed to fail the test even if the test should be succeeding. 

Does it get the final return value of the command, gsutil -q stat?